web/src/view/nav/route.ts

```diff
@@ -6,6 +6,7 @@
  */
 export enum Route {
   HOME = 'app/index',
+  PARTY = 'app/party',
   LECTURES = 'app/lectures',
   PROJECTS = 'app/projects',
   PLAYGROUND = 'app/playground',
```

Comment:
Should we delete these unused routes? In general, I don't know if we need to delete code we're not using.

Reply:
I think we can delete them. I don't think there should be any issue with that